internal/kuberesource/parts.go

```diff
@@ -25,10 +25,13 @@ func ContrastRuntimeClass(platform platforms.Platform) (*RuntimeClassConfig, err
 		return nil, fmt.Errorf("getting default runtime handler: %w", err)
 	}
 
+	// Consists of the default VM memory, 70MiB for the Kata shim and 100MiB for qemu overhead.
```

Comment:
```suggestion
	// Consists of the default VM memory, 70MiB for the Kata shim and 100MiB for kernel overhead.
```

nit: I think this is more correct.

Reply:
Also, shouldn't we address this in `DefaultMemoryInMegaBytes` directly, as it applies for all containers?

Reply:
Ah, I get the semantics now. I think we could add to the comment of `DefaultMemoryInMegaBytes` that it corresponds to Kata's `default_memory`.

Reply:
No, this is actually for the host overhead qemu has (total qemu usage minus quest VM size).